content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,207 @@
+
+See the [GatewayClass](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1.GatewayClass)
+reference for a full definition of this API kind.
+
+### Gateway {#api-kind-gateway}
+
+A Gateway describes an instance of traffic handling infrastructure. It defines a network endpoint
+that can be used for processing traffic, i.e. filtering, balancing, splitting, etc. for backends
+such as a Service. For example, HTTP traffic sent to a Kubernetes Service by a cloud load balancer
+or an in-cluster proxy server.
+
+A minimal Gateway resource example:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: example-gateway
+spec:
+  gatewayClassName: example-class
+  listeners:
+  - name: http
+    protocol: HTTP
+    port: 80
+```
+
+In this example, an instance of traffic handling infrastructure is programmed to listen for HTTP
+traffic on port 80. Since the `addresses` field is unspecified, an address or hostname is assigned
+to the Gateway by the controller. This address is used as a network endpoint for processing traffic
+of backend network endpoints defined in routes.
+
+See the [Gateway](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1.Gateway)
+reference for a full definition of this API kind.
+
+### HTTPRoute {#api-kind-httproute}
+
+The HTTPRoute kind specifies routing behavior of HTTP requests from a Gateway listener to backend network
+endpoints. For a Service backend, an implementation may represent the backend network endpoint as a Service
+IP or the backing Endpoints of the Service. An HTTPRoute represents configuration that is applied to the
+underlying Gateway implementation. For example, defining a new HTTPRoute may result in configuring additional
+traffic routes into a cloud load balancer or in-cluster proxy server.
+
+A minimal HTTPRoute example:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: example-httproute
+spec:
+  parentRefs:
+  - name: example-gateway
+  hostnames:
+  - "www.example.com"
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /login
+    backendRefs:
+    - name: example-svc
+      port: 8080
+```
+
+In this example, HTTP traffic from Gateway `example-gateway` with the Host: header set to `www.example.com`
+and the request path specified as `/login` will be routed to Service `example-svc` on port `8080`.
+
+See the [HTTPRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1.HTTPRoute)
+reference for a full definition of this API kind.
+
+## Request flow
+
+Here is a simple example of HTTP traffic being routed to a Service by using a Gateway and an HTTPRoute:
+
+{{< figure src="/docs/images/gateway-request-flow.svg" alt="A diagram that provides an example of HTTP traffic being routed to a Service by using a Gateway and an HTTPRoute" class="diagram-medium" >}}
+
+_GatewayClass is ommitted in the figure for brevity purposes._
```

Comment:
I think GatewayClass just doesn't really have a place in representing traffic flow

```suggestion
```

Reply:
@michaelbeaumont It's important to get the perspective of someone new to Gateway API, so I really appreciate your review. GatewayClass could have a place in representing traffic flow but not in this example. For instance, `gatewayclass.spec.parametersRef` may contain Gateway config parameters that would influence traffic flow, e.g. use a cloud load balancer to load balance traffic across multiple instances that represent a Gateway.

Reply:
I agree that in this particular example, GatewayClass doesn't really have a place. I think this sentence can just be removed.